README.rst

```diff
@@ -125,16 +115,16 @@ Disconnect  If a file named "xxx_down.bat" exist in the config folder
 Registry Values affecting the OpenVPN GUI operation
 ***************************************************
 
-All OpenVPN GUI reg-values are located below the following key:
-HKEY_LOCAL_MACHINE\SOFTWARE\OpenVPN-GUI\
+All OpenVPN GUI registry values are located below the
+*HKEY_LOCAL_MACHINE\\SOFTWARE\\OpenVPN-GUI\\* key
 
 The follow keys are used to control the OpenVPN GUI
 
 config_dir
-    configuration file directory, defaults to "C:\Program Files\OpenVPN\config"
+    configuration file directory, defaults to *C:\\Program Files\\OpenVPN\\config*
```

Comment:
May be change the description "configuration file directory" to "system-wide configuration file directory" ? And/or clarify that we also allow user-specific config directory which is hardwired to OpenVPN\config in user's home